docs/navigation-configuration.md

```diff
@@ -155,10 +155,10 @@ To define all subsequent nodes, use the category label:
 {
   category: 'Links',
   externalLink: {
-    url: 'http://www.luigi-project.io',
+    url: 'http://www.luigi-project.io/docs/navigation-configuration',
```

Comment:
The changes in this file are not needed. 

This part is a documentation of the feature for **externalLink**s. It is just an example configuration, not something that will affect the website build. 
For this example we want to have it simple and abstract. Very custom links like docs/navigation-configuration add confusion here. This is more like www.example.com kind of an example link that we have used.

Reply:
Since we add another node with the same URL I thought it might be better, I can remove this or should I use something like www.example.com?

Reply:
You have already added the new node here **website/docs/static/public/navigation-nodes.json**.
That JSON file you changed will change how the documentation website is built. 

This _navigation-configuration.md_ here does not affect how the website is built. This is just a simple code example of how the feature is supposed to work. 

I would just revert this MD file here to what it was before and then it should be okay. No changes are needed here.

Reply:
I am reading the whole file, yes makes sense 👍 🤔